skins/shared/color.scss

```diff
@@ -5,6 +5,7 @@ $color-red: #f00;
 
 // Default colors
 $color-links-default: #006CB0;
+$color-local-nav-default: #006CB0;
```

Comment:
color-buttons was using links-default as a default
that is why I used as well
as community header is to have same default as button